### ISSUE TITLE
check-tag-names doesn't know @internal

### ISSUE THREAD

**Issue by Shinigami92** [gajus/eslint-plugin-jsdoc#859]
## Expected behavior

`check-tag-names` should not warn on valid tag `@internal`

```js
/**
 * @internal
 */
```

## Actual behavior
> Invalid JSDoc tag name "internal". eslint [jsdoc/check-tag-names](https://github.com/gajus/eslint-plugin-jsdoc#eslint-plugin-jsdoc-rules-check-tag-names)

## ESLint Config

https://github.com/faker-js/faker/blob/877ac76b2811c948c560fee4fa76908b142557d1/.eslintrc.js

## Environment

- Node version: v16.13.2
- ESLint version v8.11.0
- `eslint-plugin-jsdoc` version: 38.0.6
